### PR TITLE
Added toggle/copy code feature for all elements

### PR DESCRIPTION
**Pull request by nimishjain77**
- Added HTML & Typescript code and copy button to copy the code
- Need to add `position: relative` to the parent component and use `toggleComponent` as a Child component

### CODE REVIEW

File: libs/ptg-ui-web-components-angular/src/index.ts [hunk 1]
```diff
@@ -1 +1 @@
-export * from './lib/ptg-ui-web-components-angular';
```
Comment (ZalawaHarsha) [inline libs/ptg-ui-web-components-angular/src/index.ts:1]: why this change is needed ?

**Comment by nimishjain77** [comment]
@lokeshdaiya @lokeshdaiya-yash Please review and merge the PR, So that we can use Copy Code in further components.

File: libs/ptg-ui-web-components-angular/src/index.ts (hunk 1 above)
Comment (nimishjain77) [reply libs/ptg-ui-web-components-angular/src/index.ts:1]: It was a wrong path while importing modules from `ptg-ui-web-components-angular`
All of the modules are imported inside `lib/web-components-angular.ts` file.

Please have a look at the below screenshot:

image
